src/app/rundown/components/rundown/rundown.component.ts

```diff
@@ -23,18 +24,22 @@ export class RundownComponent implements OnInit, OnDestroy {
 
   public rundown?: Rundown
   private rundownSubscription?: SubscriptionLike
+  private readonly logger: Logger
 
   constructor(
     private readonly route: ActivatedRoute,
     private readonly rundownService: RundownService,
     private readonly rundownStateService: RundownStateService,
-    private readonly dialogService: DialogService
-  ) {}
+    private readonly dialogService: DialogService,
+    logger: Logger
+  ) {
+    this.logger = logger.tag('RundownComponent')
```

Comment:
I know we can't use the `[class].name` in production, from a similar comment made by Lindved. But is there any other way we can ensure the string here always matches the class it is in. Maybe a Linting rule? It is just very easy thing to miss chainging, should a class/component be renamed.

Reply:
This really is a small thing that is not worth spending that much time/resources on.